.github/workflows/linux_builds.yml

```diff
@@ -215,6 +217,8 @@ jobs:
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}-${{github.sha}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}-${{github.ref}}
             ${{github.job}}-${{env.GODOT_BASE_BRANCH}}
+        continue-on-error: true # if the cache load fails continue build (very common issue)
+
```

Comment:
```suggestion
        continue-on-error: true # If the cache fails to load, continue the build (very common issue).
```

Reply:
I'd just remove the comment instead of repeating it 10 times.

Reply:
fixed 🤣